### ISSUE TITLE
Better resiliency for corrupted config file

### ISSUE THREAD

**Issue by mairin** [containers/podman-desktop#1728]
**TLDR; We should be able to gracefully handle when the Podman Desktop settings.json config file is corrupted.**

I was mucking with the Podman Desktop config file at ~/.local/share/containers/podman-desktop/configuration/settings.json trying to trigger the welcome screen to pop up to review
https://github.com/containers/podman-desktop/pull/1726#pullrequestreview-1344321798 ....

And I must have mucked it up and corrupted the file. From that point on, I got the initializing black screen of death when trying to run Podman Desktop (after a a fresh checkout, yarn build, yarn watch):

![Image](https://user-images.githubusercontent.com/799683/225767646-b4d7c124-6eae-486e-8298-c283b29c3207.png)

![Image](https://user-images.githubusercontent.com/799683/225767657-526a788b-6c43-4e54-9845-b8b6c22edfcd.png)

I had first tried to clear the welcome.version property (incorrectly I guess!), then I tried emptying the file. Removing the file entirely gets things back into a working state.

**Comment by themr0c** [comment]
I had the issue today while testing https://github.com/containers/podman-desktop/pull/3794.

For now it deserves at leas an entry in troubleshooting.
